README.md

```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
 shell script so that you don't need python installed when loading the
 dashboards.
 
+
+Create a new dashboard
+----------------------
+
+If you added support for a new protocol in Packetbeat or a module in
+Metricbeat, it would be nice to create a dedicated Kibana dashboard to 
+prove the functionality of your data.
+The Kibana dashboards are saved in a special index in Elasticsearch. By default
+it's .kibana, but it can be configured to anything else.
+
+The first step, in creating your own Kibana dashboard, would be to load 
+all the current dashboards in Kibana using the
+load.sh script. Then, create the Kibana dashboard using Kibana functionalities
+and then download all the dashboards using the save/kibana_dump.py script. 
+
+Note: Make sure you are using the latest Kibana version to create and download
+the dashboards.
+
+Before executing the save/kibana_dump.py script, make sure you have python and virtualenv
+installed:
+
+        # Prepare the envirement
+        virtualenv env
+        . env/bin/activate
+        pip install -r requirements.txt
+
+        # Download all Kibana dashboards to your host
+        python kibana_dump.py --url 'http://localhost:9200' --dir output
+
+where url points to the Elasticsearch URL, dir is the directory where to place
```

Comment:
where url points to the Elasticsearch URL, and dir is the directory where you want to save the Kibana dashboards.